node/cn/api_backend.go

```diff
@@ -208,6 +208,7 @@ func (b *CNAPIBackend) GetEVM(ctx context.Context, msg blockchain.Message, state
 	vmError := func() error { return nil }
 
 	context := blockchain.NewEVMContext(msg, header, b.cn.BlockChain(), nil)
+	vmCfg.UseOpcodeComputationCost = true
```

Comment:
Is this change necessary?

Reply:
The `GetEVM` method is only used in the `DoCall` method, which is only used in `api_public_blockchain.go.` The `DoCall` method is used for `EstimateGas` and `EstimateComputationCost.` Like issue #1035, this change was added to prevent the estimation value from being changed in advance by not adding this change.

Reply:
`DoCall` can be called by three different APIs: `Call`, `DoEstimateGas`, `EstimateComputationCost`. Among them,  `DoEstimateGas`, `EstimateComputationCost` already configure `UseOpcodeComputationCost`. The other API, `Call`, intentionally disabled `UseOpcodeComputationCost` to provide unlimited computation for view functions of smart contracts. Therefore, this change was not appropriate for the current API policy.

Reply:
I got the point. I deleted that line, so please take another look. Thanks.